topicgenerator/templates/topic-generator.config

```diff
@@ -1,7 +1,4 @@
 # This is a template topic generator config file for prod. Modify it for different environments.
 [dockstore]
 server-url: https://dockstore.org/api
-token: YOUR_ADMIN_OR_CURATOR_DOCKSTORE_TOKEN
-
-[ai]
-openai-api-key: YOUR_OPENAI_API_KEY
```

Comment:
It seems since the openai code is present, could leave `openai-api-key` and just note that it is optional